rplugin/python3/molten/outputchunks.py

```diff
@@ -178,6 +179,7 @@ class Output:
     status: OutputStatus
     success: bool
     old: bool
+    start_time: datetime
```

Comment:
In addition to the start_time, you should store the end time when the cell completes. You can determine when the cell completes [here](https://github.com/benlubas/molten-nvim/blob/8346bba69e0de96278dad2038e9be74605908b7d/rplugin/python3/molten/moltenbuffer.py#L217).

Then you can update the logic above to use this value if it's not `None` or use the current time (this would imply that the cell is still running).

Reply:
I incorporated part of this change.  I tried implementing the seconds part of your suggestion (Then you can update the logic above to use this value if it's not None or use the current time (this would imply that the cell is still running)), but the time doesn't update after the initial execution.  Not sure why the output buffer doesn't re-render, maybe you might have a better idea.

Reply:
I'm seeing the output timer change. Just that there is no 'live' runtime, it just shows "Running".

That should be fixable

Reply:
Ah what you said is what I meant to describe.  I'll work on this some more.